packages/mco/components/modals/app-manage-policies/manage-policy-view.tsx

```diff
@@ -82,8 +83,19 @@ const getAggregatedDRInfo = (
       assignedOn: formatTime(
         getLatestDate([acc.assignedOn, drpc?.metadata?.creationTimestamp])
       ),
+      volumeReplicationGroup:
+        acc.volumeReplicationGroup ||
+        drpc?.metadata?.annotations?.[
+          'drplacementcontrol.ramendr.openshift.io/is-cg-enabled'
+        ] === 'true',
```

Comment:
```suggestion
        getAnnotations(drpc)?.[IS_CG_ENABLED_ANNOTATION] === 'true';
```
declare constant for 'drplacementcontrol.ramendr.openshift.io/is-cg-enabled' in mco/constants/disaster-recovery.ts

Reply:
done